client/packages/common/src/intl/locales/es/inventory.json

```diff
@@ -2,7 +2,6 @@
   "label.finalised": "Finalizado",
   "heading.description": "Descripción",
   "label.batch": "Lote",
-  "label.location": "Ubicación",
```

Comment:
Would the translation tool removed it automatically if you would have left it in?

Reply:
only if `yarn i18n-unused-remove` was run - we don't remove automatically afaik